README.md

```diff
@@ -31,39 +31,112 @@ Even though it trains part of the neural network models, NNtrainer requires quit
 
 ## Components
 
-### NeuralNetwork
+### Supported Layers
 
-This is the component which controls neural network layers. Read the configuration file ([Iniparser](https://github.com/ndevilla/iniparser) is used to parse the configuration file.) and constructs Layers including Input and Output Layer, according to configured information by the user.
-The most important role of this component is to activate forward / backward propagation. It activates inferencing and training of each layer while handling the data properly among them. There are properties to describe network model as below:
+This component defines Layers which consist of Neural Network Model. Layers has own properties to be set.
 
-- **_Type:_** Network Type - Regression, k-NN, NeuralNetwork
-- **_Layers:_** Name of Layers of Network Model
-- **_Learning\_rate:_** Learning rate which is used for all Layers
-- **_Decay\_rate:_** Rate for Exponentially Decayed Learning Rate
-- **_Epoch:_** Max Number of Training Iteration.
-- **_Optimizer:_** Optimizer for the Network Model - sgd, adam
-- **_Activation:_** Activation Function - sigmoid , tanh
-- **_Cost:_** Cost Function -
-      mse(mean squared error), cross (cross entropy)
-- **_Model:_** Name of Model. Weight Data is saved in the name of this.
-- **_minibach:_** mini batch size
-- **_beta1,beta2,epsilon:_** hyper parameters for the adam optimizer
+ | Keyword | Layer Name | Description |
+ |:-------:|:---:|:---|
+ |  conv2d | Convolution 2D |Convolution 2-Dimentional Layer |
+ |  pooling2d | Pooling 2D |Pooling 2-Dimentional Layer. Support average / max / global average / global max pooing |
+ | flatten | Flatten | Flatten Layer |
+ | fully_connected | Fully Connected | Fully Connected Layer |
+ | input | Input | Input Layer.  This is not requies always. |
+ | batch_normalization | Batch Normalization Layer | Batch Normalization Layer. |
+ | loss layer | loss layer | hidden from users |
+ | activation | activaiton layer | hidden from users. but set by layer property | 
```

Comment:
Currently Activation layer is exposed to support activation after batch normalization

Reply:
fixed.